charts/privatebin/Chart.yaml

```
@@ -6,7 +6,7 @@ name: privatebin
 home: https://privatebin.info/
 icon: https://raw.githubusercontent.com/PrivateBin/assets/master/images/preview/icon.png
 type: application
-version: 0.15.0
+version: 0.15.1
```

adding features so bumping minor version

```suggestion
version: 0.16.0
```